### PR TITLE
🍱 Migrate missing samples.

### PR DESCRIPTION
**Pull request by matthiasrohmer**
- AMPHTML Ads Hello World
- amp-date-display
- AMPHTML Ads vs non-AMP ads
- Shopping Cart
- SeatMap Multiple Selection (requires SSR, therefore marked as `draft: true` temporarily)

### CODE REVIEW

File: examples/source/e-commerce/Shopping_Cart.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
+<!---
+
+disablePlayground: true
```
Comment (sebastianbenz) [inline examples/source/e-commerce/Shopping_Cart.html]: re-enable the playground

File: examples/source/0.introduction/AMPHTML_Ads_Hello_World.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    h1 {
+      color: red;
+    }
+  </style>
+</head>
+<!-- ## Body -->
+<!-- -->
+<body>
+  <!--
+    Inside the body, AMPHTML ads are only allowed to use a restricted set of HTML and AMPHTML elements as described in the [spec](https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/amp-a4a-format.md).
```
[the remark this answers is not in the document]
Comment (matthiasrohmer) [reply examples/source/0.introduction/AMPHTML_Ads_Hello_World.html]: WDYT of using `[AMPHTML ads]({{ g.doc('/content/amp-dev/documentation/guides-and-tutorials/learn/a4a_spec.md').url.path }})`? Wasn't quite sure if it works, but it does and would make the links fail-safe (does the document exist, has been moved, ...).

[the remark this answers is not in the document]
Comment (matthiasrohmer) [reply examples/source/0.introduction/AMPHTML_Ads_Hello_World.html]: Would of course also work for the components reference.

File: examples/source/e-commerce/Shopping_Cart.html (hunk 1 above)
Comment (matthiasrohmer) [reply examples/source/e-commerce/Shopping_Cart.html]: It has been disabled on abe.com, but don't know why...

Comment (sebastianbenz) [reply examples/source/e-commerce/Shopping_Cart.html]: I don't know either - I think it's fine to re-enable

File: examples/source/0.introduction/AMPHTML_Ads_Hello_World.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (sebastianbenz) [reply examples/source/0.introduction/AMPHTML_Ads_Hello_World.html]: perfect

[the remark this answers is not in the document]
Comment (matthiasrohmer) [reply examples/source/0.introduction/AMPHTML_Ads_Hello_World.html]: Nice!

**Pull request merged**